src/using-guacamole.md

```diff
@@ -82,20 +82,24 @@ again or swipe left across the screen.
 
 The Guacamole menu provides options for:
 
-* Navigating back to the home screen
+* [Navigating back to the home screen](client-user-menu)
```

Comment:
Have you considered putting these shortcuts in the same order as the sections below?

Reply:
That does sound sensible.

Reply:
There.